include/translated-term.php

```diff
@@ -139,11 +139,17 @@ public function delete_translation( $id ) {
 	 *
 	 * @since 1.2
 	 *
+	 * @param string $alias Alias for $wpdb->terms table
 	 * @return string join clause
 	 */
-	public function join_clause() {
+	public function join_clause( $alias = '' ) {
```

Comment:
It's just a nitpicking, but `$alias = 't'` would spare the `if (empty ( $alias )` check below.

Reply:
Kept the similar coding style as seen in the posts join method. Thanks for merging!

Reply:
@chesio You are right. I will change that.